ibc-core/ics24-host/types/src/error.rs

```diff
@@ -9,6 +9,35 @@ use ibc_primitives::prelude::*;
 use ibc_primitives::proto::Error as ProtoError;
 use prost::DecodeError as ProstError;
 
+/// Errors that originate from host implementations.
+#[derive(Debug, Display)]
+pub enum HostError {
```

Comment:
Does it make sense to provide some helper functions like we did in #1335? For example, `invalid_data<T: ToString>(description: T)`. Since `HostError` will be used by integrators, such functions could be helpful.

Reply:
[96b4797](https://github.com/cosmos/ibc-rs/pull/1331/commits/96b4797f800c6b903bd66a389c4a4e74d3e037f7) and [ee93e43](https://github.com/cosmos/ibc-rs/pull/1331/commits/ee93e43c7f389efc0d50f1499c63f18466be4a51)